x-pack/plugin/ilm/src/internalClusterTest/java/org/elasticsearch/xpack/ilm/ILMMultiNodeWithCCREnabledIT.java

```diff
@@ -41,12 +43,12 @@
 import static org.hamcrest.Matchers.equalTo;
 
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0, numClientNodes = 0)
-public class ILMMultiNodeIT extends ESIntegTestCase {
+public class ILMMultiNodeWithCCREnabledIT extends ESIntegTestCase {
```

Comment:
I think we don't have to rename this as it's not meant to test anything CCR related

Reply:
++